UnityProject/Assets/Scripts/Objects/VisibleBehaviour.cs

```diff
@@ -36,6 +36,9 @@ public class VisibleBehaviour : NetworkBehaviour
 		closetHandler
 	};
 
+	public GameObject[] ignoreSpriteUpdates;
+	private SpriteRenderer[] ignoredSpriteRenderers;

```

Comment:
Why not just make this public and drag the spriterenderers into the inspector? You just need to drag the whole game object and it will take the sprite renderer from it